src/components/Logo/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from "react";
+import styled from "styled-components";
+import { minDevice } from "../../assets/styles/breakpoints";
+
+import logoBlack from "../../assets/img/logo-black.svg";
+import logoWhite from "../../assets/img/logo-white.svg";
```

Comment:
This is a matter of one's prefferd style, but I can share main:
I use an empty line between imports from outside libraries (whatever we installed in node modules) our own files we've created and style files (when I use css). Then the imports would look like that:
```suggestion
import React, { FC } from "react";
import styled from "styled-components";

import { minDevice } from "../../assets/styles/breakpoints";
import logoBlack from "../../assets/img/logo-black.svg";
import logoWhite from "../../assets/img/logo-white.svg";
```